web/src/js/components/MainView.tsx

```diff
@@ -11,6 +11,9 @@ export default function MainView() {
     const hasSelection = useAppSelector(
         (state) => !!state.flows.byId[state.flows.selected[0]],
     );
+    const isMultipleFlowsSelection = useAppSelector(
```

Comment:
I don't quite understand why `isMultipleFlowsSelection` exists? In most parts of the UI you use `selectedFlows.length > 1` but then in this one part you use this flag instead. Either consistently track `isMultipleFlowsSelection` or consistently imply it via `selectedFlows.length > 1`. Or am I misunderstanding what's happening?

Reply:
you're right, having both sounds confusing! I've removed `isMultipleFlowsSelection`